### PR TITLE
Version v0.0.14

### PR DESCRIPTION
**Pull request by aprilrieger**
New workflow_dispatch inputs:
- baseTarget
- webTarget

Base, web, worker & solr:
- if clause added to each retag action with proper registry tagging
- if clause added to each build and push action

Removed some workflow_dispatch inputs since the max workflow_dispatch inputs we can have (for now) are 10. I had to get creative in replacing the if clauses from the boolean options to if the baseTarget, solrTarget, webTarget, and/or workerTarget are not empty (ex: `if: ${{ inputs.webTarget != '' }}`), so that it would trigger a workflow block to run with just the target set. Removing the need for the workflow_dispatch inputs:
```yaml
worker:
  required: false
  type: boolean
```
As well as setting base, solr, web, and/or worker to true (ex: `worker: true`) in the associated reusable workflow.

### CODE REVIEW

**Comment by aprilrieger** [comment]
Before merging I need to add the updated version.
After merging I need to cut a new release.

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -2,6 +2,9 @@ name: "Rspec for Rails Apps"
 on:
   workflow_dispatch:
     inputs:
+      base:
```
Comment (orangewolf) [inline .github/workflows/test.yaml:5]: can we use *Target for all of these? that way there is only one variable (baseTarget in this case) that works the same across linters, tests and build. It accomplishes the same thing but with one fewer concepts

**Review by orangewolf — COMMENTED** [review]
I really like where you are headed. one further simplification that I think is worth doing (but could be talked out of)

File: .github/workflows/test.yaml (hunk 1 above)
Comment (aprilrieger) [reply .github/workflows/test.yaml]: @orangewolf Let me know what you think of my changes, I appreciate the suggestions to simplify!

File: .github/workflows/lint.yaml [hunk 2]
```diff
@@ -2,24 +2,50 @@ name: "Lint for Rails Projects"
 on:
   workflow_dispatch:
     inputs:
+      baseTarget:
+        description: "Used to enable the linter to run on the base image"
+        required: false
+        type: boolean
```
Comment (orangewolf) [inline .github/workflows/lint.yaml:8]: I think there was a misunderstanding here. These should be the same as in build. That means they should be strings, not booleans and then below we should use the same `${{ inputs.baseTarget != '' }}` to check them. That way there is only one format across all the files for targets.

**Review by orangewolf — CHANGES_REQUESTED** [review]

**Comment by aprilrieger** [comment]
@orangewolf This is ready for a review please. I am getting this polished to cut a new release for use with ATLA-Hyku's CI setup.

**Review by orangewolf — APPROVED** [review]

**Pull request merged**